AGENTS.md

```
@@ -1,168 +1,118 @@
 
-## Build/Lint/Test Commands
+## 👤 Agent Guidelines
 
-### Shell Script Validation
+### Workflow Order
 
-```bash
-bash -n cli.sh              # Syntax check single script
-bash -n generate.sh         # Syntax check single script
-bash -n cli.sh generate.sh  # Check all scripts
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)